### PR TITLE
update hub and tag examples in readme

### PR DESCRIPTION
**Pull request by linsun**
with example of specifying your own hub and tag

### CODE REVIEW

**Comment by clyang82** [comment]
the test is using the default tag and hub. I think it works. what is your case?

**Comment by linsun** [comment]
@clyang82 i didn't update the test, not sure why it is failing. I was just going through the readme to make it clear.

**Comment by linsun** [comment]
@clyang82 I had issues with corporate firewall so my machine can't access GCR at work (thanks to @sdake) so I had to use released images from docker hub for now before that is resolved. I've updated the PR to make that clear.

**Review by clyang82 — APPROVED** [review]

**Comment by clyang82** [comment]
rebase should help.

File: README.md [hunk 1]
```diff
@@ -155,6 +155,14 @@ This is currently required if any mTLS is used. In future other Spifee implement
...
 
 ```bash
+# use the default HUB=gcr.io/istio-release, TAG=master-latest-daily defined in global.yaml
+iop istio-system citadel $IBASE/security/citadel
+```
+
+If you want to use a different `HUB` and `TAG`, the following example shows how to install istio 1.3.0's citadel:
+
+```bash
+export HUB=docker.io/istio TAG=1.3.0
```
Comment (sdake) [inline README.md]: this looks fine - although inconsistent with our developer wiki.  In our developer wiki, we recommend setting HUB and TAG and ISTIO.  See: https://github.com/istio/istio/wiki/Preparing-for-Development#setting-up-environment-variables

I guess I don't know what $IBASE is :)

**Review by sdake — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (linsun) [reply README.md]: If I set per developer wiki, it won't work in this scenario.  I believe TAG istio is wrong and confusing also.

**Pull request merged**